py_proxy/views.py

```diff
@@ -14,8 +16,54 @@ def status(_request):
     return response.Response(status_int=200, status="200 OK", content_type="text/plain")
 
 
+@view.view_config(route_name="robots", http_cache=(86400, {"public": True}))
+def robots(request):
+    """Serve robots.txt file."""
+    return response.FileResponse(
+        "py_proxy/static/robots.txt", request=request, content_type="text/plain"
+    )
+
+
+@view.view_config(route_name="favicon", http_cache=(86400, {"public": True}))
+def favicon(request):
+    """Serve favicon.ico file."""
+    return response.FileResponse(
+        "py_proxy/static/favicon.ico", request=request, content_type="image/x-icon"
+    )
```

Comment:
Are these favicon and robots views necessary? I think the `config.add_static_view(name="static", path="static")` means they'll be served automatically

Reply:
You are correct that they are served at /static/robots.txt however, they need to be served from the root endpoint (/robots.txt) so that line by itself will not work for either of these files.